conan/tools/system/pip_manager.py

```diff
@@ -23,6 +25,18 @@ def generate(self):
         env.prepend_path("PATH", self.bin_dir)
         env.vars(self._conanfile).save_script(self.env_name)
 
+    def _create_venv(self):
```

Comment:
What about relying on a user configuration pointing to the python interpreter when not found in path?